WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -58,6 +58,9 @@ struct PointOfSaleDashboardView: View {
         .animation(.easeInOut(duration: Constants.connectivityAnimationDuration), value: viewModel.showsConnectivityError)
         .background(Color.posPrimaryBackground)
         .navigationBarBackButtonHidden(true)
+        .sheet(item: $totalsViewModel.cardPresentPaymentOnboardingViewModel) { viewModel in
```

Comment:
From a usability point of view, I dismissed the Stripe onboarding web view at least a few times by accidentally tapping on the background. This forced me to re-enter Stripe onboarding details. I don't know if there's an easy way to prevent it on Swift UI.

Reply:
Good point, it could be a frustrating experience having filled out a bunch of info in the Stripe form and then the form is accidentally closed from dragging or tapping outside of the modal. In https://github.com/woocommerce/woocommerce-ios/pull/14185/commits/f754d0a7ecb6a9f5fbbd2f19fc7e09b64eb63fc2, I added `interactiveDismissDisabled` to the sheet so that it can't be dismissed by these built-in gestures.

Reply:
Thanks for the fix!